Sources/Nimble/Expectation.swift

```diff
@@ -6,7 +6,7 @@ internal func expressionDoesNotMatch<T, U>(_ expression: Expression<T>, matcher:
     msg.to = toNot
     do {
         let pass = try matcher.doesNotMatch(expression, failureMessage: msg)
-        if msg.actualValue == "" {
+        if "" == msg.actualValue {
```

Comment:
I believe Yoda conditions is not appropriate for Swift: https://realm.github.io/SwiftLint/yoda_condition.html